File: app/src/main/java/com/tutv/android/domain/Episode.java
package com.tutv.android.domain;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity
public class Episode {

    @PrimaryKey
    @ColumnInfo(name = "episode_id")
    private int id;

    private String name;
    @ColumnInfo(name = "num_episode") private int numEpisode;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getNumEpisode() {
        return numEpisode;
    }

    public void setNumEpisode(int numEpisode) {
        this.numEpisode = numEpisode;
    }
}

File: app/src/main/java/com/tutv/android/domain/Episode.kt
package com.tutv.android.domain

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.ForeignKey
import androidx.room.PrimaryKey

@Entity(foreignKeys = [ForeignKey(
                entity = Season::class,
                parentColumns = ["season_id"],
                childColumns = ["season_id"])
        ]
)
class Episode {
    @PrimaryKey @ColumnInfo(name = "episode_id") var id = 0
    var name: String? = null
    @ColumnInfo(name = "num_episode") var numEpisode = 0
    @ColumnInfo(name = "season_id") var seasonId = 0
    var loggedInUserViewed: Boolean? = null
}